source/includes/pipelines/_02-status.md

```diff
@@ -0,0 +1,32 @@
+## Get pipeline status
+
+```shell
+
+$ curl 'http://ci.example.com/go/api/pipelines/pipeline1/status' \
+       -u 'username:password'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=utf-8
+```
+
+```json
+{
+  "locked":false,
+  "paused":false,
+  "schedulable":true
+}
+```
+
+The pipeline status allows users to check the pipeline status like paused, locked and schedulable.
```

Comment:
Change to "..allows users to check if the pipeline is paused, locked and schedulable..."